YIF.Core.Domain/Repositories/LectorRepository.cs

```diff
@@ -43,7 +44,7 @@ public async Task<IEnumerable<LectorDTO>> Find(Expression<Func<Lector, bool>> pr
             throw new NotImplementedException();
         }
 
-        public Task<LectorDTO> Get(string id)
+        public async Task<LectorDTO> Get(string id)
```

Comment:
why are you added async to no implemented method?

Reply:
Fixed